src/components/search/PatientSearch.jsx

```diff
@@ -8,6 +8,18 @@ import patientUtil from '../../domain/patient/patientUtil';
 import PatientSearchForm from './PatientSearchForm';
 import DataGrid from "../grid/DataGrid";
 
+const formatIdentifier = (identifier) => {
```

Comment:
Hmm... this is problematic, because it is very Malawi-specific and should live in the the Malawi workflow module (like the IdentifierFilter), and not in react-components itself.

Is there a way this could be put in with the IdentifierFilter somehow?  Could there a method in that component that does this normalization?

Reply:
@mogoodrich done!